File: spring-batch-plus-sample/spring-batch-plus-single-class-reader-processor-writer/src/test/java/com/navecorp/spring/batch/plus/sample/RunSamplesTest.java
package com.navecorp.spring.batch.plus.sample;

import org.junit.jupiter.api.Test;

public class RunSamplesTest {

	@Test
	void all() throws Exception {
		com.navercorp.spring.batch.plus.sample.flux.callback.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.flux.readerprocessorwriter.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.flux.readerwriter.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.iterable.callback.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.iterable.readerprocessorwriter.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.iterable.readerwriter.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.iterator.callback.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.iterator.readerprocessorwriter.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.iterator.readerwriter.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.simple.callback.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.simple.readerprocessorwriter.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.simple.readerwriter.BatchApplication.main(new String[0]);
	}
}

File: spring-batch-plus-sample/spring-batch-plus-single-class-reader-processor-writer-kotlin/src/test/kotlin/com/navercorp/spring/batch/plus/sample/RunSamplesTest.kt
package com.navercorp.spring.batch.plus.sample

import org.junit.jupiter.api.Test

class RunSamplesTest {

    @Test
    fun all() {
        com.navercorp.spring.batch.plus.sample.flux.callback.main()
        com.navercorp.spring.batch.plus.sample.flux.readerprocessorwriter.main()
        com.navercorp.spring.batch.plus.sample.flux.readerwriter.main()
        com.navercorp.spring.batch.plus.sample.iterable.callback.main()
        com.navercorp.spring.batch.plus.sample.iterable.readerprocessorwriter.main()
        com.navercorp.spring.batch.plus.sample.iterable.readerwriter.main()
        com.navercorp.spring.batch.plus.sample.iterator.callback.main()
        com.navercorp.spring.batch.plus.sample.iterator.readerprocessorwriter.main()
        com.navercorp.spring.batch.plus.sample.iterator.readerwriter.main()
        com.navercorp.spring.batch.plus.sample.simple.callback.main()
        com.navercorp.spring.batch.plus.sample.simple.readerprocessorwriter.main()
        com.navercorp.spring.batch.plus.sample.simple.readerwriter.main()
    }
}
